app/serializers/crate.js

```diff
@@ -10,4 +12,23 @@ export default class CrateSerializer extends ApplicationSerializer {
 
     return super.extractRelationships(...arguments);
   }
+
+  normalizeQueryResponse(_store, _modelClass, payload) {
+    // We don't want existing relationships overwritten by results with null values.
+    // See: https://github.com/rust-lang/crates.io/issues/10711
+    if (payload.crates) {
+      payload = {
+        ...payload,
```

Comment:
is that assignment doing anything? seems like the `payload` variable isn't being used afterwards

Reply:
yeah, `payload.crates` should be enough!